ui-components/src/locales/es.json

```diff
@@ -1,750 +1,1264 @@
 {
```

Comment:
Why all the changes to this one? Aren't all the strings already translated in bloom?

Reply:
Two reasons I'm spotting: 
1. Over time it seems that general.json and es.json started to diverge. I expect people made changes to general.json without updating es.json, so some of the fields didn't match up in the upstream file. 
2. For some strings the translations generated by Google translate and those from the upstream es.json file are different.

Reply:
I'm just wondering if at some point we should try to reconcile it with upstream or ask them if they will be re-translating at some point and we can pull theirs once they have "official" ones. Once ours are finalized we can contribute Bengali and Arabic back upstream, but what do we do here?

Reply:
I think we should commit our version for now and just pick our version each time if there is a merge conflict. 

One problem with reconciling the file or asking them to retranslate is that our file has already started to diverge from theirs. This is because the pattern is to define all strings here even if they are site specific. Personally, I think it would be great to have more separation between strings that are associated with ui-components and strings that are defined & used in sites to avoid these issues. Same issues with contributing Bengali and Arabic upstream. We already have strings defined here that don't need to be pushed upstream.